### PR TITLE
Add integration tests for Get and Scan with a Cassandra secondary index.

### PR DESCRIPTION
**Pull request by feeblefakie**
I added simple integration tests for Get and Scan with a Cassandra secondary index.
It should be merged after #137 is merged.
(or, maybe separate #137 into C* part and Cosmos part and consolidate the C* part with this PR?)

### CODE REVIEW

**Review by yito88 — DISMISSED** [review]
Thank you! LGTM 👍 
I'll add the same tests to CosmosIntegrationTest.

**Review by brfrn169 — APPROVED** [review]
LGTM!

File: src/integration-test/java/com/scalar/db/storage/cassandra/CassandraIntegrationTest.java [hunk 1]
```diff
@@ -1050,6 +1054,71 @@ public void put_IncorrectPutGiven_ShouldThrowIllegalArgumentException() {
...
+    assertThat(actual.get(1).getValue(COL_NAME4))
+        .isEqualTo(Optional.of(new IntValue(COL_NAME4, 1)));
+    assertThat(actual.get(2).getValue(COL_NAME1))
+        .isEqualTo(Optional.of(new IntValue(COL_NAME1, 3)));
+    assertThat(actual.get(2).getValue(COL_NAME4))
+        .isEqualTo(Optional.of(new IntValue(COL_NAME4, 0)));
+  }
+
+  @Test
+  public void scan_ScanGivenForNonIndexedColumn_ShouldThrow() {
```
Comment (feeblefakie) [inline src/integration-test/java/com/scalar/db/storage/cassandra/CassandraIntegrationTest.java:1112]:
```suggestion
  public void scan_ScanGivenForNonIndexedColumn_ShouldThrowIllegalArgumentException() {
```

**Comment by yito88** [comment]
@feeblefakie Sorry, the base branch was changed. I think the test will work after #145.

**Pull request merged**